mozdownload/scraper.py

```diff
@@ -46,7 +46,7 @@
                       'mac': r'mac',
                       'mac64': r'mac(64)?',
                       'win32': r'win32',
-                      'win64': r'win64-x86_64'}
```

Comment:
Thank you for the patch David, but as I have mentioned on IRC yesterday I have a nearly ready patch to go too. Something else which makes it a bit more complicate is that we cannot hard-switch as you did here. We have to support both naming schemes because Thunderbird didn't flip that yet, and we would not be able to download older Firefox 64 bit builds. I hope you have nothing against it when I get my PR finished instead?